Modelica/Magnetic/FluxTubes/Material/HysteresisEverettParameter/BaseData.mo

```diff
@@ -3,9 +3,9 @@ record BaseData "Base Data Record"
   extends Modelica.Icons.Record;
   parameter SI.MagneticFieldStrength Hsat=100
     "Hysteresis region between -Hsat .. Hsat";
-  parameter Real M(final unit="1")=0.95
+  parameter Real M(final unit="T")=0.95
     "Related to saturation value of magnetization";
-  parameter Real r(final unit="1")=0.55
+  parameter Real r(final unit="T-(1/2)")=0.55
```

Comment:
See: https://github.com/modelica/ModelicaSpecification/pull/3439 for this new unit-string, added after Modelica Language 3.6.